infra/concourse/build/Dockerfile.lint

```diff
@@ -35,6 +35,11 @@ RUN wget https://github.com/segmentio/terraform-docs/releases/download/v0.6.0/te
     mv terraform-docs* /usr/local/bin/terraform-docs && \
     chmod 0755 /usr/local/bin/terraform-docs
 
-RUN wget https://raw.githubusercontent.com/antonbabenko/pre-commit-terraform/master/terraform_docs.sh && \
-    mv terraform_docs.sh /usr/local/bin/terraform_docs.sh && \
-    chmod 0755 /usr/local/bin/terraform_docs.sh
```

Comment:
This is removed because terraform_docs is being added into the helpers folder of each module until such time as each module is updated to use the lint image to perform lint checks.

Reply:
Let's leave this in place. It should not be an issue if modules add the script locally and this image will be valid for the modules which assume the script is provided.

Reply:
Added it back in.